client/package.json

```diff
@@ -63,7 +63,7 @@
       "^(?!.*\\.(js|jsx|mjs|cjs|ts|tsx|css|json)$)": "<rootDir>/config/jest/fileTransform.js"
     },
     "transformIgnorePatterns": [
-      "[/\\\\]node_modules[/\\\\].+\\.js$",
+      "node_modules/@mozilla/(?!glean)/",
```

Comment:
@Guyzeroth This looks to me like _previously_ all `*.js` in `node_modules` should have been ignored, and _now_ only directories in `node_modules/@mozilla` except `glean` are ignored. In other words, now everything in `node_modules` is transformed except the non-glean directories in `node_modules/@mozilla/`.

Reply:
```suggestion
      "node_modules/(?!@mozilla/glean)/",
```